stores/healthcareProfessionalsStore.ts

```diff
@@ -12,6 +17,17 @@ export const useHealthcareProfessionalsStore = defineStore(
         const healthcareProfessionalsData: Ref<HealthcareProfessional[]>
         = ref([])
         const selectedHealthcareProfessionalId: Ref<string> = ref('')
+        const healthcareProfessionalSectionFields = {
```

Comment:
The code would likely be less complex if we make this a single object `ref` instead of an object of a bunch of `ref`s.